src/scss/components/_modal-waitlist.scss

```diff
@@ -26,4 +26,18 @@
       width: calc(100% - #{$grid-gutter-width-lg});
     }
   }
+
+  .modal-content {
+    max-width: calc(100% - #{$grid-gutter-width});
+    @include media-breakpoint-up('lg') {
+      width: 100%;
+    }
+  }
+
+  .text-danger {
+    top: -#{$grid-gutter-width/2};
+    @include media-breakpoint-up('lg') {
+      top: -#{$grid-gutter-width-lg/2};
```

Comment:
this element is not absolute nor relative, so we should use margin-top property, in this way we avoid the slide down of the fields when there is an error

Reply:
updated with margin-top